grafana/sunandmoon.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: 1
+
+datasources:
+  - name: Sun and Moon
+    type: grafana-sunandmoon-datasource
```

Comment:
I haven't been able to fully test this. It could be wrong. :)

Reply:
I found via https://github.com/fetzerch/grafana-sunandmoon-datasource/blob/master/provisioning/datasources/default.yml that it's 
```
   type: fetzerch-sunandmoon-datasource
   jsonData:
     latitude: 48.0
     longitude: 10.0
     ```

Reply:
Excellent! Thanks @longzheng